modules/services/holochain-conductor.nix

```
@@ -31,9 +31,11 @@ in
         ${pkgs.hpos-config-into-keystore}/bin/hpos-config-into-keystore \
           < $HPOS_CONFIG_PATH > /tmp/holo-keystore 2> /tmp/holo-keystore.pub
         export HOLO_KEYSTORE_HCID=$(cat /tmp/holo-keystore.pub)
-        ${pkgs.envsubst}/bin/envsubst < ${pkgs.writeTOML cfg.config} \
-          | ${pkgs.holo-update-conductor-config}/bin/holo-update-conductor-config \
-          $STATE_DIRECTORY/conductor-config.toml
+        if [ ! -f $STATE_DIRECTORY/conductor-config.toml || ! $HPOS_DEVELOPMENT_MODE ]; then
```

```suggestion
        if [[ ! -f $STATE_DIRECTORY/conductor-config.toml || $HPOS_DEVELOPMENT_MODE -ne "true" ]]; then
```
``||`` logical operator is only avail inside `[[ ]]`